.travis.yml

```diff
@@ -57,9 +61,20 @@ before_install:
 - sudo apt-get update
 - sudo apt-get install -y --force-yes r-base
 - R --version
-#install gcc
-- sudo apt-get install -y --force-yes  gcc
-- gcc --version
+###### necessary for running the dynamically linked GenomicsDB binaries in src/test/resources/large
+#Install lcov and MPICH
+- sudo apt-get -y install lcov mpich zlib1g-dev libssl-dev
+- sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
```

Comment:
Unlikely - it is maintained by Canonical (and not an arbitrary user) - if it does change, we will submit a patch ( we use the same PPA in the TileDB/GenomicsDB Travis setup)